### PR TITLE
Replaced winston with log4js

### PR DESCRIPTION
**Pull request by askmon**
- Removed winston
- Added log4js
- Added logstash udp support

### CODE REVIEW

**Comment by askmon** [comment]
@lucas-coelho :point_up:

File: package.json [hunk 1]
```diff
@@ -1,16 +1,13 @@
 {
   "name": "tq1-logger",
-  "version": "0.0.4",
+  "version": "0.0.5",
```
Comment (felipesabino) [inline package.json]: The API changed so change major versions instead of minor one. It could even be 1.0.0

File: src/logger.coffee [hunk 2]
```diff
@@ -1,54 +1,50 @@
...
-      when 'info' then consoleConfig.level = 'info'
-      when 'warn' then consoleConfig.level = 'warn'
-      when 'error' then consoleConfig.level = 'error'
-
-    logger = new winston.Logger transports: [new winston.transports.Console consoleConfig]
+  @startLogger: (options, mainConsole) ->
+
+    logCategory = if options.name? then options.name else "TQ1 - Logs"
+
+    if options.logstashHost? and options.logstashPort? and options.name?
```
Comment (felipesabino) [inline src/logger.coffee]: @askmon The docs and these checks are not clear for those using the component. We should change here with one of the two following options:
1. Add a `options.logstashEnabled`, check this variable instead and log an error if any of these mandatory variables are not present (this should also be in the docs)
2. Update the docs explaining that in order to use logstash as an output, all these params are mandatory

File: package.json (hunk 1 above)
Comment (askmon) [inline package.json]: done

File: src/logger.coffee (hunk 2 above)
Comment (askmon) [inline src/logger.coffee]: Updated docs

**Comment by jessicafernandes209** [comment]
👍

**Comment by felipesabino** [comment]
:+1:

**Pull request merged**